### PR TITLE
Feature/elasticsearch

### PR DESCRIPTION
**Pull request by blongden**

### CODE REVIEW

File: elasticsearch/2.4/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# ELASTICSEARCH 2.4
+
+```Dockerfile
+FROM quay.io/continuouspipe/elasticsearch:2.4
```
Comment (kierenevans) [inline elasticsearch/2.4/README.md]: `FROM quay.io/continuouspipe/elasticsearch2.4:stable`

File: docker-compose.yml [hunk 2]
```diff
@@ -178,6 +178,13 @@ services:
     depends_on:
       - external_redis
 
+  elasticsearch:
+    build:
+      context: ./elasticsearch/2.4/
```
Comment (andytson-inviqa) [inline docker-compose.yml]: I'd recommend dropping this 2.4 off. If we want different versions, we'll template it the same way the mongodb and mysql ones are done

Comment (andytson-inviqa) [inline docker-compose.yml]: I mean, storing it all in ./elasticsearch/ rather than ./elasticsearch/2.4/

Comment (blongden) [inline docker-compose.yml]: Oh, just using Dockerfile-2.4 etc?

Comment (andytson-inviqa) [inline docker-compose.yml]: You can leave that as Dockerfile for now unless you want to add other versions

Comment (andytson-inviqa) [inline docker-compose.yml]: the other Dockerfile-*'s are only committed as CP doesn't have a pre-build step that can run build.sh

Comment (blongden) [inline docker-compose.yml]: Gotcha.

**Comment by kierenevans** [comment]
In advance of the PR being merged, I have created https://quay.io/repository/continuouspipe/elasticsearch2.4 and the CP build robot has been added with write permissions.

**Pull request merged**